Personal Access Token documentation updates (#572)
rudderlabs/rudderstack-docs: transformations/rudderstack-transformation-api.mdx

<div class="warningBlock">

> It is mandatory to generate your RudderStack personal access token to be able to make any API calls. For more information, refer to the <a href="https://rudderstack.com/docs/rudderstack-api/personal-access-tokens/">Personal Access Tokens</a> guide.

Reviewer: Should change "generate your RudderStack personal access token to be able to make any API calls" to "generate a personal access token to make any API calls"

> It is mandatory to generate your RudderStack personal access token to be able to make any API calls. For more information, refer to the <a href="https://rudderstack.com/docs/rudderstack-api/personal-access-tokens/">Personal Access Tokens</a> guide.

Author: ```suggestion
You need to generate a personal access token to make any API calls. For more information, refer to the <a href="https://rudderstack.com/docs/rudderstack-api/personal-access-tokens/">Personal Access Tokens</a> guide.
```
